### DISCUSSION TITLE
i can't retrieve Members list (objects browser)

### DISCUSSION THREAD

**Discussion by ngrulier** [codefori/vscode-ibmi#731] [General]
Hello, 
I'm Nicolas

when i want retrieve the member list, this one stay empty. 

on the log, we can see the DB2 request didn't load the source members list

when i read the temporary file with a SQL client ( DBeaver), i saw the file was not empty.

I saw all DB2 queries had no return data :

```
SELECT * FROM QSYS2.ASP_INFO
{
    "code": 0,
    "signal": null,
    "stdout": "DB2>",
    "stderr": ""
}

/home/GRN: LC_ALL=EN_US.UTF-8 system "call QSYS/QZDFMDB2 PARM('-d' '-i')"
select SYSTEM_VALUE_NAME, CURRENT_NUMERIC_VALUE from QSYS2.SYSTEM_VALUE_INFO where SYSTEM_VALUE_NAME = 'QCCSID'
{
    "code": 0,
    "signal": null,
    "stdout": "DB2>",
    "stderr": ""
`}
```

i work on V7R4.

![image](https://user-images.githubusercontent.com/6436659/177309548-0a917db7-d193-4663-8fdc-9055283e2beb.png)

**Comment by worksofliam** [comment]
@ngrulier Looks similar to this issue: https://github.com/halcyon-tech/vscode-ibmi/issues/562

**Reply by ngrulier** [reply]
Hi,
Thank you for your answer. 
I've disable SQL mode and that's work now

But My CCSID is not 65535. The ccsid of my USRPRF  is 500 . 
I've typed [system "DSPJOB"] in a bash 

my profile PASE is using bash instead of qsh.

We are using V7R4 TR5 

in my ssh session i wrote that : 
 > system -i "call QSYS/QZDFMDB2 PARM('SELECT * FROM ILEDITOR.O_FPX5r5UZ')"
and i can read the temporary file (see below)
So, the included command ( system "call QSYS/QZDFMDB2 PARM('-d' '-i' '-t')"
SELECT * FROM ILEDITOR.O_FPX5r5UZ)  doesn't work.

![image](https://user-images.githubusercontent.com/6436659/178444358-0c4c6b90-2ccb-4f9f-b166-188ee21df4a9.png)

In order to discover how the QZDFMDB2 program works , i've searched the ibm documentation but i've found nothing.
Where can i find it ? If you kwow.
